ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -74,10 +75,16 @@ class LocalAssemblerData
         assert(local_matrix_size == ShapeFunction::NPOINTS * NUM_NODAL_DOF);
     }
 
-    void assemble(double const /*t*/, std::vector<double> const& local_x) override
+    void assemble(std::size_t const id,
```

Comment:
For the meaning of `id` see my comment above.
I wrote `executeMemberOnDereferenced()` s.t. it passes the id as the first argument, because for future use it might be handy, and because the other `execute...()` methods also pass it. Therefore the `id` parameter is present here and I use it instead of `_element.getID()`.